bitcoincash/houses/cashfusion.md

```diff
@@ -0,0 +1,21 @@
+---
+title: cash fusion                                          
```

Comment:
This is the title of the page so it should be `Cash Fusion`. When I said all lowercase I meant the file name.